components/icon-buttons.jsx

```diff
@@ -1,14 +1,22 @@
 var React = require('react');
 
 var IconButtons = React.createClass({
+  propTypes: {
+    children: React.PropTypes.object.isRequired
+  },
   render: function() {
+    let nonNullChildren = this.props.children.filter((child) => {
+      return child !== null;
+    });
+    let numColSpan = 12 / nonNullChildren.length; // Bootstrap uses 12-col layout
```

Comment:
This could result in floating point numbers if you don't use Math.floor or Math.ceil